MTM-46430 Review user & developer documentation for upcoming GA release (#560)
SoftwareAG/c8y-docs: content/reference/smartrest-one-bundle/smartrest-real-time-notifications.md

layout: redirect
---

> All available real-time notification endpoints and channels of the {{< product-c8y-iot >}} platform are also available in a SmartREST syntax. See [Real-time notification API](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Real-time-notification-API) to understand the general functionality of the [Bayeux protocol](https://docs.cometd.org/current/reference/#_concepts_bayeux_protocol) and get an overview of our available endpoints and channels for real-time notifications.

Reviewer: ```suggestion
All available real-time notification endpoints and channels of the {{< product-c8y-iot >}} platform are also available in a SmartREST syntax.
See the [Real-time notification API](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Real-time-notification-API) to understand the general functionality of the [Bayeux protocol](https://docs.cometd.org/current/reference/#_concepts_bayeux_protocol) and to get an overview of our available endpoints and channels for real-time notifications.
```

> All available real-time notification endpoints and channels of the {{< product-c8y-iot >}} platform are also available in a SmartREST syntax. Please have a look at the [Real-time notifications API](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Real-time-notification-API) to understand the general functionality of the [Bayeux protocol](https://docs.cometd.org/current/reference/#_concepts_bayeux_protocol) and get an overview of our available endpoints and channels for real-time notifications.

Reviewer 2: ```suggestion
All available real-time notification endpoints and channels of the {{< product-c8y-iot >}} platform are also available in a SmartREST syntax. See [Real-time notification API](https://{{< domain-c8y >}}/api/{{< c8y-current-version >}}/#tag/Real-time-notification-API) to understand the general functionality of the [Bayeux protocol](https://docs.cometd.org/current/reference/#_concepts_bayeux_protocol) and get an overview of our available endpoints and channels for real-time notifications.
```
